src/Tracker.ts

```diff
@@ -0,0 +1,436 @@
+import { Socket } from 'dgram';
+import { createWriteStream, WriteStream } from 'fs';
+import { ConnectionTracker } from './ConnectionTracker';
+import { Protocol, SensorStatus } from './constants';
+import { IncomingAccelPacket } from './packets/IncomingAccelPacket';
+import { IncomingBatteryLevelPacket } from './packets/IncomingBatteryLevelPacket';
+import { IncomingCalibrationFinishedPacket } from './packets/IncomingCalibrationFinishedPacket';
+import { IncomingErrorPacket } from './packets/IncomingErrorPacket';
+import { IncomingGyroPacket } from './packets/IncomingGyroPacket';
+import { IncomingHandshakePacket } from './packets/IncomingHandshakePacket';
+import { IncomingHeartbeatPacket } from './packets/IncomingHeartbeatPacket';
+import { IncomingMagnetometerAccuracyPacket } from './packets/IncomingMagnetometerAccuracy';
+import { IncomingPongPacket } from './packets/IncomingPongPacket';
+import { IncomingRawCalibrationDataPacket } from './packets/IncomingRawCalibrationDataPacket';
+import { IncomingRotationDataPacket } from './packets/IncomingRotationDataPacket';
+import { IncomingSensorInfoPacket } from './packets/IncomingSensorInfoPacket';
+import { IncomingSignalStrengthPacket } from './packets/IncomingSignalStrengthPacket';
+import { IncomingTapPacket } from './packets/IncomingTapPacket';
+import { IncomingTemperaturePacket } from './packets/IncomingTemperaturePacket';
+import { IncomingCorrectionDataPacket } from './packets/inspection/IncomingCorrectionDataPacket';
+import { IncomingFusedIMUDataPacket } from './packets/inspection/IncomingFusedIMUDataPacket';
+import { IncomingRawIMUDataPacket } from './packets/inspection/IncomingRawIMUDataPacket';
+import { OutgoingHandshakePacket } from './packets/OutgoingHandshakePacket';
+import { OutgoingPingPacket } from './packets/OutgoingPingPacket';
+import { OutgoingSensorInfoPacket } from './packets/OutgoingSensorInfoPacket';
+import { PacketWithSensorId } from './packets/Packet';
+import { parse } from './packets/PacketParser';
```

Comment:
Please make an index file in the packet folder. this is awfull

Reply:
Soon, making the packets a library when this PR is merged

Reply:
Ok then